apps/api/src/hyp3_api/routes.py

```diff
@@ -111,6 +111,13 @@ def __call__(self, errors):
 )
 
 
+# TODO tests
+@app.route('/costs', methods=['GET'])
+@openapi
+def costs_get():
+    return jsonify(handlers.get_costs())
```

Comment:
I'm not convinced a handler is necessary here. We could make this look like the /openapi.json handler above. It would mean importing dynamo.jobs.COSTS here, though, and I'm open to arguments that routes.py shouldn't need to know anything about the dynamo package.